crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -252,4 +305,18 @@ mod tests {
         let error_message = result.unwrap_err().to_string();
         assert_eq!("Failed to parse value invalid to felt", error_message);
     }
+
+    #[test]
+    fn test_default_account_name_import() {
+        let accounts_file = Utf8PathBuf::from("tests/data/accounts/accounts.json");
+
+        let (account, account_id) = super::get_import_name(None, &accounts_file);
+        let (account_2, account_id_2) = super::get_import_name(None, &accounts_file);
+
+        assert_eq!(account, "account-1");
+        assert_eq!(account_id, Some(1));
+
+        assert_eq!(account_2, "account-2");
+        assert_eq!(account_id_2, Some(2));
+    }
```

Comment:
For some reason, this test and all other tests in the module are not captured.

Reply:
Please base on https://github.com/foundry-rs/starknet-foundry/blob/master/crates/sncast/tests/e2e/account/import.rs while adding tests.